src/server/cluster/cluster_family.cc

```diff
@@ -598,12 +599,24 @@ inline CommandId::Handler HandlerFunc(ClusterFamily* se, EngineFunc f) {
 
 #define HFUNC(x) SetHandler(HandlerFunc(this, &ClusterFamily::x))
 
+namespace {
+namespace Acl {
```

Comment:
namespaces should be all lowercase, and for consistency, I suggest using `acl` everywhere.
and you probably won't need `namespace Cat = AclCategory;` then.

Reply:
+1

Reply:
Honestly, I really like the change, it's so much simpler now without the nested namespaces. Moreover, I removed the unnamed namespaces since `constexpr` implies `const` which implies `static` which has `internal linkage`